doc/lua_api.md

```diff
@@ -4111,9 +4111,10 @@ On some specific cases, server translation could be useful. For example, filter
 a list on labels and send results to client. A method is supplied to achieve
 that:
 
-`minetest.get_translated_string(lang_code, string)`: Translates `string` using
-translations for `lang_code` language. It gives the same result as if the string
-was translated by the client.
+`minetest.get_translated_string(lang_code, string)`: resolves translations in
+the given string just like the client would, using the translation files for
```

Comment:
I think you forgot the single quotes here: `string`

Reply:
I'd say no, as string is a noun and not referencing verbatim code specifically

Reply:
I guess then it has to say ”the given string `string`” or something like that?

Rationale: Without this patch the text used to reference the `string` argument and with this patch it does not.